packages/ui-components/src/components/text-field/text-field.scss

```
@@ -0,0 +1,98 @@
+@import '../../assets/styles/globals';
+
+$defaultInputHeight: 35px;
+
+.label,
+.error {
+	@include kv-font-label-small-uppercase-semibold
+}
+
+.label {
+	color: kv-color('text');
+}
+
+.error {
+	color: kv-color(error);
+}
+
+.flex-align-center {
+	display: flex;
+	align-content: center;
+}
+
+.text-container {
+	margin-left: $spacing-2x;
+
+	.text-elem {
+		width: 100%;
+		height: 16px;
+		margin-top: $spacing;
+
+		.flex-align-center {
+			display: flex;
+			align-content: center;
+		}
+
+		color: kv-color(primary, base);
```

always try to call the function with strings 

```suggestion
		color: kv-color('primary', 'base');
```

---

Please validate that different states have the appearance defined in [ZeroHeight](https://zeroheight.com/1b10805cd/p/41c9b5-text-field/b/87b9f3)